src/api/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -63,6 +64,24 @@ private ItemStackUtils()
          */
     }
 
+    /**
+     * Getter for the predicate describing food in minecolonies.
+     * @return it.
+     */
+    public static Predicate<ItemStack> getFoodPredicate()
+    {
+        return itemStack -> !ItemStackUtils.isEmpty(itemStack) && itemStack.getItem() instanceof ItemFood;
```

Comment:
you can make this a method and usa it by reference :) no need to call a method to return a second one if you don't use it dynamically

Reply:
I need to construct a predicate at those places, if I don't get it like that I will have the work again of constructing a predicate